pep-0458.txt

```diff
@@ -369,6 +369,12 @@ generate cryptographic keys, and manage a TUF repository.
 
 __ https://github.com/theupdateframework/tuf/blob/v0.11.1/docs/TUTORIAL.md#how-to-create-and-modify-a-tuf-repository
 
+In standard operation the *bin-n* metadata will be created and signed as new
+distributions are uploaded to PyPI. However, there will also need to be a
+one-time online initialization mechanism which to create and sign *bin-n*
```

Comment:
```suggestion
one-time online initialization mechanism to create and sign *bin-n*
```

Reply:
good catch, thanks! Addressed in rebased & force-pushed patches.